package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bokeh/jupyter_bokeh",
-  "version": "3.0.0-dev1",
+  "version": "3.0.0-dev2",
```

Comment:
It should be `3.0.0-dev.2` to keep it consistent with semver and other bokeh's npm packages.

Reply:
FWIW the server regex validator that the semver project links to in its docs validates the above version, which is why i used it.